src/main/java/com/squareup/kotlinpoet/KModifier.kt

```diff
@@ -61,7 +61,11 @@ enum class KModifier(
   // Type modifiers.
   IN("in", Target.VARIANCE_ANNOTATION),
   OUT("out", Target.VARIANCE_ANNOTATION),
-  VARARG("vararg", Target.PARAMETER);
+  VARARG("vararg", Target.PARAMETER),
+
+  // Multiplatform modules.
+  HEADER("header", Target.CLASS),
+  IMPL("impl", Target.CLASS);
```

Comment:
This is also valid in a function and probably a property

Reply:
Is there any documentation btw?

Reply:
Uhh. I don't think there's much if any. I can point you to a bunch of example usage though.

Reply:
Sure, that'd be great

Reply:
https://github.com/JakeWharton/Reagent has the referenced atomic thing from the issue.

The others are private. Can invite later when I'm not mobile. Beyond that you can search "Kotlin multiplatform Gradle" and find a few others' samples.

Reply:
Thanks!